library/Icinga/Web/RememberMeCookie.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/* Icinga Web 2 | (c) 2020 Icinga Development Team | GPLv2+ */
+
+namespace Icinga\Web;
+
+/**
+ * Class RememberMeCookie
```

Comment:
See the other comment regarding PHPDoc for classes. Fix the first line and remove the package tag.